lib/tasks/test.rake

```diff
@@ -16,4 +16,36 @@ namespace :test do
     t.verbose = true
     t.warning = false
   end
+
+  namespace :syntax do
+    desc "Validate selected template snapshots against ksvalidator with EL7 profile"
+    task :kickstart_el7 do
+      Dir.glob('test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/*Kickstart*').each do |file|
+        puts "Validating #{file}"
+        `ksvalidator --version RHEL7 "#{file}"`
+        abort("invalid kickstart: #{file}") unless $?.success?
+      end
+    end
+
+    desc "Validate selected template snapshots against ksvalidator with EL8 profile"
+    task :kickstart_el8 do
+      Dir.glob('test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/*Kickstart*').each do |file|
+        puts "Validating #{file}"
+        `ksvalidator --version RHEL8 "#{file}"`
```

Comment:
Given EL7 and EL8 differs just by this one parameter, would it be worth to have common kickstart taks, that would accept the EL version param and these two would just `invoke` that task with correct version parameter?

Reply:
I have no idea how to execute tests for both el7 and el8 if you write just `rake syntax:kickstart` tho.